vendor/flarum/tags/less/common/TagLabel.less

```
@@ -60,3 +59,42 @@
     }
   }
 }
+@media (max-width: 767px){
+    .item-tags{
+        overflow: hidden ;
+        text-overflow: ellipsis ;
+        white-space: nowrap ;
+        border-radius: @border-radius @border-radius @border-radius @border-radius;
```

Nếu như cả 4 cái cùng 1 giá trị thì em chỉ cần
```suggestion
        border-radius: @border-radius;
```